src/content/docs/en/installation/getting.mdx

```diff
@@ -32,16 +27,16 @@ Ultramarine Linux is distributed in various formats available for download from
   sites other than our website or FOSSTorrents
 </Alert>
 
-### Atomic
+### Atomic Preview
 
-Ultramarine Linux also comes in Atomic versions. Ultramarine Atomic works similarly to Fedora Silverblue, CoreOS, and other OSTree-based operating systems. This variant is made for users who want a stable, immutable system that is easy to recover from.
-We see immutability as the future of Linux systems. Ideally, this will become the standard edition in the future.
+With Ultramarine 41 we are now providing a formal preview of our atomic variant. Ultramarine Atomic is based on Fedora bootc, but we intend to switch to our own base container (based on Fedora packages) in the future.
+Bootc, and atomicity generally still has some papercuts, but things move fast. We see this as the future of Linux systems. Ideally, this will become our standard edition in the future.

```

Comment:
> Ideally, this will become our standard edition in the future.

I don't think we ever agreed on this, did we?

Reply:
we did, and this language was already here

Reply:
I thought I opposed because UM as a UX friendly distro shouldn't encourage immutability?

Reply:
there is a reason it's still in preview, UX isn't there yet but it is being worked on

Reply:
Well if not UX then DX then, immutability definitely doesn't work well with DX.